ext/redhat/logrotate

```diff
@@ -1,9 +1,23 @@
-/var/log/puppetlabs/puppet.log {
+# Generic logrotate config for Red Hat systems. Works with both official
+# Fedora and RHEL packages as well as PuppetLabs distributions.
+
+/var/log/puppetlabs/masterhttp.log /var/log/puppet/masterhttp.log {
+  compress
   missingok
   notifempty
-  create 0644 puppet puppet
+  # permissions/owner must be aligned with puppet/defaults.rb
+  create 0660 puppet puppet
```

Comment:
Assuming that and amending the change, it looks like this will work with SELinux which will do proper transition as well.